FAIMS/FAIMS3
# feat: display template used

## JIRA Ticket

[BSS-327](https://jira.csiro.au/browse/BSS-327)

## Description

Added template used to survey details tab.

## Proposed Changes

- Added template used to survey details tab.

## How to Test

1. Create a new survey using a template.
2. Open the created survey.
3. Click on the details tab and verify that the template used is displayed.

## Additional Information

Behaviour currently is that the field appears empty if no template was used.

## Checklist
